10_date_timestamps_and_lies.md

```diff
@@ -145,31 +143,10 @@ df.select(to_date($"lastUpdate").as("lastUpdate"))
 // +----------+-----+
 ```
 
-and
```

Comment:
This doesn't add much to a previously printed schema. Maybe some comment would be useful there.